addons/mass_mailing/controllers/main.py

```diff
@@ -195,3 +195,15 @@ def send_feedback(self, mailing_id, res_id, email, feedback, token):
                 record.sudo().message_post(body=_("Feedback from %(email)s: %(feedback)s", email=email, feedback=feedback))
             return bool(records)
         return 'error'
+
+    @http.route('/mailing/report/unsubscribe', type='http', website=True, auth='public')
```

Comment:
if this route is only accessible to internal users, can't you set `auth='user'`?

Reply:
The idea was that the user doesn't have to be logged in to disable it, much like digest reports since it might be on another device and there isn't really a need to log in security wise